ROI harness minitest fails on missing bits2fasm.py

```
+ prjxray/utils/bits2fasm.py --verbose design.bits 
./runme.sh: line 60: prjxray/utils/bits2fasm.py: No such file or directory
Makefile:14: recipe for target 'build/design.json' failed
```
commit 2ba905130ab9f27c37ccfc5afcbe66be6a0479e0 removes the bits2fasm tool. It looks like we should use bit2fasm instead

--- a/minitests/roi_harness/runme.sh
+++ b/minitests/roi_harness/runme.sh
@@ -57,7 +57,7 @@ test -z "$(fgrep CRITICAL vivado.log)"
 
 ${XRAY_BITREAD} -F $XRAY_ROI_FRAMES -o design.bits -z -y design.bit
 ${XRAY_SEGPRINT} -zd design.bits >design.segp
-${XRAY_DIR}/utils/bits2fasm.py --verbose design.bits > design.fasm
+${XRAY_DIR}/utils/bit2fasm.py --verbose design.bit > design.fasm
 ${XRAY_DIR}/utils/fasm2frames.py design.fasm design.frm
 python3 ../create_design_json.py --design_info_txt design_info.txt --design_txt design.txt --pad_wires design_pad_wires.txt > design.json
 

